src/dpi.rs

```diff
@@ -1,80 +1,83 @@
-//! DPI is important, so read the docs for this module if you don't want to be confused.
+//! Pixel types and UI scaling information.
 //!
-//! Originally, `winit` dealt entirely in physical pixels (excluding unintentional inconsistencies), but now all
-//! window-related functions both produce and consume logical pixels. Monitor-related functions still use physical
-//! pixels, as do any context-related functions in `glutin`.
+//! ## Why should I care about UI scaling?
 //!
-//! If you've never heard of these terms before, then you're not alone, and this documentation will explain the
-//! concepts.
+//! Modern computer screens don't have a consistent relationship between resolution and size.
+//! 1920x1080 is a common resolution for both desktop and mobile screens, despite mobile screens
+//! normally being less than a quarter the size of their desktop counterparts. What's more, neither
+//! desktop nor mobile screens are consistent resolutions within their own size classes - common
+//! mobile screens range from below 720p to above 1440p, and desktop screens range from 1080p to 5K
```

Comment:
I'm not sure sub-1080p desktop/laptop screens are actually uncommon yet. For a random example, see the primacy of 1366x768 at https://gs.statcounter.com/screen-resolution-stats/desktop/worldwide.

Reply:
Oh wow, they're even more common than 1080p screens. Good catch